_pages/PrashanthGanoji.yaml.txt

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
your file is a .txt extension file.
Correct it saving as .yaml
Sometimes the text editor software we use adds a .txt by default, check this.